modules/utxo-lib/test/integration_local_rpc/generate/regtestNode.ts

```diff
@@ -41,7 +41,7 @@ function getDockerParams(network: Network): DockerImageParams {
     case utxolib.networks.dashTest:
       return dockerImage('dashpay/dashd:0.16.1.1', 'dashd');
     case utxolib.networks.dogecoinTest:
-      return dockerImage('williamqinbitgo/dogeimage:1.14.5-v3', 'dogecoind');
+      return dockerImage('williamqinbitgo/dogeimage:1.14.5-v2', 'dogecoind');
```

Comment:
why?

Reply:
It turns out that the `v3` version I published is actually broken, and that the regtest fixtures had been generated incorrectly (from an earlier PR with one of my commits). This commit should fix the generation.

Reply:
I updated the v3 docker image to be working and roughly 3 GB smaller in size so there's no need to revert anymore

Reply:
ideally you would have called it `-v4` then ;)